src/fixate/drivers/dmm/helper.py

```diff
@@ -67,3 +67,6 @@ def diode(self, low_current=True, high_voltage=False):
 
     def reset(self):
         raise NotImplementedError
+
+    def get_serial(self):
```

Comment:
I'm torn on if this should be NotImplementedError or return "". We might not be able to get a serial number from all drivers and for some, it might not make sense. I suppose the argument in favor of keeping it NotImplementedError would be to force the driver implementation to be explicit. It can alwa

Reply:
Some of the other drivers simply pass rather than raising an error.